public-api/src/main/java/org/pmiops/workbench/cdr/model/AchillesResult.java

```diff
@@ -0,0 +1,192 @@
+        this.stratum4 = stratum4;
+    }
+    public AchillesResult stratum4(String val) {
+        this.stratum4 = val;
+        return this;
+    }
+
+    @Column(name="stratum_5")
+    public String getStratum5() {
+        return stratum5;
+    }
+    public void setStratum5(String stratum5) {
+        this.stratum5 = stratum5;
+    }
+    public AchillesResult stratum5(String val) {
+        this.stratum5 = val;
+        return this;
+    }
+
+    @Transient
+    public String getStratum5Name() {
+        return stratum5Name;
+    }
+    public void setStratum5Name(String stratum5Name) {
+        this.stratum5Name = stratum5Name;
+    }
+    public AchillesResult stratum5Name(String val) {
+        this.stratum5Name = val;
+        return this;
+    }
+
+    @Column(name="count_value")
+    public Long getCountValue() {
+        return countValue;
+    }
+    public void setCountValue(Long countValue) {
+        this.countValue = countValue;
+    }
+    public AchillesResult countValue(Long val) {
+        this.countValue = val;
+        return this;
+    }
+
+    @Override
+    public String toString() {
+        return "AchillesResult{" +
+                "id=" + id +
+                ", analysisId=" + analysisId +
+                ", analysis=" + analysis +
+                ", stratum1='" + stratum1 + '\'' +
+                ", stratum2='" + stratum2 + '\'' +
+                ", stratum3='" + stratum3 + '\'' +
+                ", stratum4='" + stratum4 + '\'' +
+                ", stratum5='" + stratum5 + '\'' +
+                ", countValue=" + countValue +
+                ", stratum5Name='" + stratum5Name + '\'' +
+                '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        AchillesResult that = (AchillesResult) o;
+
+        if (id != null ? !id.equals(that.id) : that.id != null) return false;
+        if (analysisId != null ? !analysisId.equals(that.analysisId) : that.analysisId != null) return false;
+        if (analysis != null ? !analysis.equals(that.analysis) : that.analysis != null) return false;
+        if (stratum1 != null ? !stratum1.equals(that.stratum1) : that.stratum1 != null) return false;
+        if (stratum2 != null ? !stratum2.equals(that.stratum2) : that.stratum2 != null) return false;
+        if (stratum3 != null ? !stratum3.equals(that.stratum3) : that.stratum3 != null) return false;
+        if (stratum4 != null ? !stratum4.equals(that.stratum4) : that.stratum4 != null) return false;
+        if (stratum5 != null ? !stratum5.equals(that.stratum5) : that.stratum5 != null) return false;
+        if (countValue != null ? !countValue.equals(that.countValue) : that.countValue != null) return false;
+        return stratum5Name != null ? stratum5Name.equals(that.stratum5Name) : that.stratum5Name == null;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
nit: same here..use java.util.Objects like so:
return Objects.hash(id, analysisId, analysis, stratum1, stratum2, stratum3, stratum4, stratum5, countValue, stratum5Name);

Reply:
Hopefully this will get rid of the Codacy complains about these.